bin/disco_rds.py

```diff
@@ -55,6 +56,16 @@ def get_parser():
     parser_cleanup_snapshots.set_defaults(mode="cleanup_snapshots")
     parser_cleanup_snapshots.add_argument('--age', dest='days', required=False,
                                           help='Minimum age of Snapshots to expire', type=int, default=30)
+
+    # clone Mode
+    parser_clone = subparsers.add_parser("clone", help="Create a new database from an existing database")
+    parser_clone.set_defaults(mode="clone")
+    parser_clone.add_argument('--env', dest='env', required=False, default=None,
+                              help='The environment containing the RDS cluster(s)')
+    parser_clone.add_argument('--source-db', dest='source_db', required=True,
```

Comment:
It would create the database under a new name. Might be useful if you want to clone the database into the same environmen as the source. 

`disco_rds.py --env foo clone --source-db txdb --source-env ci --dest-db some-other-name`

Normally clone would create a `foo-txdb` but the above command would create `foo-some-other-name`

I don't know if thats useful though